piwik-sdk/src/main/java/org/piwik/sdk/DownloadTracker.java

```diff
@@ -41,7 +43,25 @@
     public DownloadTracker(Tracker tracker) {
         this(tracker, new TrackMe());
     }
-
+    
+    public void setVersion(String version) {
+        mVersion = version;
+        System.out.println("Version is now "+version);
```

Comment:
I would suggest to use `Logy.d` here instead, as we did it [here](https://github.com/piwik/piwik-sdk-android/blob/master/piwik-sdk/src/main/java/org/piwik/sdk/Tracker.java#L723).